### ISSUE TITLE
Network: Nodes plotted very far apart

### ISSUE THREAD

**Issue by andradf** [almende/vis#3013]
using vis.js 4.19.1.
I am trying to graph a directed acyclic graph. Adding certain nodes leads to the graph nodes being initially plotted very far apart, and taking minutes if not more to stabilize.

This set of nodes/edges plots ok (still a little slow to stabilize but manageable)

nodes = [{"id":0,"label":"000","level":0},{"id":700,"label":"700","level":1},{"id":701,"label":"701","level":2},{"id":760,"label":"760","level":5},{"id":1,"label":"001","level":6},{"id":460,"label":"460","level":7},{"id":462,"label":"462","level":8},{"id":459,"label":"459","level":9},{"id":463,"label":"463","level":10},{"id":464,"label":"464","level":11},{"id":451,"label":"451","level":12},{"id":452,"label":"452","level":13},{"id":502,"label":"502","level":17},{"id":691,"label":"691","level":25},{"id":465,"label":"465","level":12},{"id":467,"label":"467","level":13},{"id":468,"label":"468","level":14},{"id":469,"label":"469","level":15},{"id":410,"label":"410","level":7},{"id":411,"label":"411","level":8},{"id":412,"label":"412","level":9},{"id":409,"label":"409","level":10},{"id":413,"label":"413","level":11},{"id":414,"label":"414","level":12},{"id":425,"label":"425","level":13},{"id":427,"label":"427","level":14},{"id":423,"label":"423","level":15},{"id":424,"label":"424","level":16},{"id":428,"label":"428","level":19},{"id":429,"label":"429","level":20},{"id":480,"label":"480","level":21},{"id":482,"label":"482","level":22},{"id":484,"label":"484","level":23},{"id":508,"label":"508","level":24},{"id":430,"label":"430","level":17},{"id":431,"label":"431","level":18},{"id":417,"label":"417","level":13},{"id":418,"label":"418","level":14},{"id":486,"label":"486","level":15},{"id":488,"label":"488","level":16},{"id":432,"label":"432","level":13},{"id":433,"label":"433","level":14},{"id":434,"label":"434","level":15},{"id":435,"label":"435","level":16},{"id":490,"label":"490","level":17},{"id":491,"label":"491","level":18},{"id":494,"label":"494","level":21},{"id":495,"label":"495","level":22},{"id":492,"label":"492","level":19},{"id":493,"label":"493","level":20},{"id":436,"label":"436","level":15},{"id":437,"label":"437","level":16},{"id":415,"label":"415","level":13},{"id":416,"label":"416","level":14},{"id":420,"label":"420","level":13},{"id":422,"label":"422","level":14},{"id":2,"label":"002","level":7},{"id":4,"label":"004","level":8},{"id":5,"label":"005","level":9},{"id":6,"label":"006","level":10},{"id":7,"label":"007","level":11},{"id":9,"label":"009","level":12},{"id":80,"label":"080","level":13},{"id":87,"label":"087","level":14},{"id":88,"label":"088","level":15},{"id":125,"label":"125","level":18},{"id":896,"label":"896","level":16},{"id":898,"label":"898","level":17},{"id":40,"label":"040","level":13},{"id":641,"label":"641","level":14},{"id":523,"label":"523","level":15},{"id":693,"label":"693","level":16},{"id":122,"label":"122","level":13},{"id":20,"label":"020","level":14},{"id":690,"label":"690","level":11},{"id":702,"label":"702","level":3},{"id":703,"label":"703","level":4},{"id":704,"label":"704","level":5}]

edges = [{"from":0,"to":1,"arrows":"to"},{"from":0,"to":700,"arrows":"to"},{"from":700,"to":701,"arrows":"to"},{"from":700,"to":701,"arrows":"to"},{"from":701,"to":702,"arrows":"to"},{"from":701,"to":760,"arrows":"to"},{"from":701,"to":760,"arrows":"to"},{"from":701,"to":760,"arrows":"to"},{"from":701,"to":760,"arrows":"to"},{"from":760,"to":1,"arrows":"to"},{"from":1,"to":2,"arrows":"to"},{"from":1,"to":410,"arrows":"to"},{"from":1,"to":460,"arrows":"to"},{"from":460,"to":462,"arrows":"to"},{"from":462,"to":459,"arrows":"to"},{"from":459,"to":463,"arrows":"to"},{"from":463,"to":464,"arrows":"to"},{"from":464,"to":465,"arrows":"to"},{"from":464,"to":451,"arrows":"to"},{"from":451,"to":452,"arrows":"to"},{"from":452,"to":502,"arrows":"to"},{"from":502,"to":691,"arrows":"to"},{"from":465,"to":467,"arrows":"to"},{"from":467,"to":468,"arrows":"to"},{"from":468,"to":469,"arrows":"to"},{"from":410,"to":411,"arrows":"to"},{"from":411,"to":412,"arrows":"to"},{"from":412,"to":409,"arrows":"to"},{"from":409,"to":413,"arrows":"to"},{"from":413,"to":414,"arrows":"to"},{"from":414,"to":420,"arrows":"to"},{"from":414,"to":415,"arrows":"to"},{"from":414,"to":432,"arrows":"to"},{"from":414,"to":417,"arrows":"to"},{"from":414,"to":425,"arrows":"to"},{"from":425,"to":427,"arrows":"to"},{"from":427,"to":423,"arrows":"to"},{"from":423,"to":424,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":430,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":424,"to":428,"arrows":"to"},{"from":428,"to":429,"arrows":"to"},{"from":429,"to":480,"arrows":"to"},{"from":480,"to":482,"arrows":"to"},{"from":482,"to":508,"arrows":"to"},{"from":482,"to":484,"arrows":"to"},{"from":484,"to":508,"arrows":"to"},{"from":508,"to":691,"arrows":"to"},{"from":430,"to":431,"arrows":"to"},{"from":431,"to":428,"arrows":"to"},{"from":417,"to":418,"arrows":"to"},{"from":418,"to":486,"arrows":"to"},{"from":486,"to":488,"arrows":"to"},{"from":488,"to":502,"arrows":"to"},{"from":432,"to":433,"arrows":"to"},{"from":433,"to":436,"arrows":"to"},{"from":433,"to":434,"arrows":"to"},{"from":434,"to":435,"arrows":"to"},{"from":435,"to":490,"arrows":"to"},{"from":490,"to":491,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":492,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":491,"to":494,"arrows":"to"},{"from":494,"to":495,"arrows":"to"},{"from":495,"to":508,"arrows":"to"},{"from":492,"to":493,"arrows":"to"},{"from":493,"to":494,"arrows":"to"},{"from":436,"to":437,"arrows":"to"},{"from":437,"to":502,"arrows":"to"},{"from":415,"to":416,"arrows":"to"},{"from":416,"to":486,"arrows":"to"},{"from":420,"to":422,"arrows":"to"},{"from":422,"to":423,"arrows":"to"},{"from":2,"to":4,"arrows":"to"},{"from":4,"to":5,"arrows":"to"},{"from":5,"to":6,"arrows":"to"},{"from":6,"to":690,"arrows":"to"},{"from":6,"to":7,"arrows":"to"},{"from":7,"to":9,"arrows":"to"},{"from":9,"to":122,"arrows":"to"},{"from":9,"to":40,"arrows":"to"},{"from":9,"to":80,"arrows":"to"},{"from":80,"to":87,"arrows":"to"},{"from":87,"to":88,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":896,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":88,"to":125,"arrows":"to"},{"from":125,"to":480,"arrows":"to"},{"from":896,"to":898,"arrows":"to"},{"from":898,"to":125,"arrows":"to"},{"from":40,"to":641,"arrows":"to"},{"from":641,"to":523,"arrows":"to"},{"from":523,"to":693,"arrows":"to"},{"from":122,"to":20,"arrows":"to"},{"from":20,"to":502,"arrows":"to"},{"from":702,"to":703,"arrows":"to"},{"from":702,"to":703,"arrows":"to"},{"from":702,"to":760,"arrows":"to"},{"from":702,"to":760,"arrows":"to"},{"from":703,"to":704,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":703,"to":760,"arrows":"to"},{"from":704,"to":1,"arrows":"to"}]

My options are 
```
{
  layout: {
    hierarchical: {
      direction: 'LR'
    }
  }
}
```

and my div is styled as `{width: 1000px; height: 1000px;}`
if I add a new node at level 16 {"id":1000, "label":"newnode", "level":16}, disconnected from anything else, it leads to the erroneous initial layout. Connecting it to the rest of the graph does not lead to better results.

**Comment by wimrijnders** [comment]
Duplicate #1735.

**Comment by wimrijnders** [comment]
Thanks for reporting. This is a long-running issue, which will be addressed centrally in #1735.

I can reproduce the issue with the data you supplied; I will be using this for testing and verification.
If I find any resolution, I hope I will remember to post it here in this issue. For now, I'm closing it.
